R/DataPull.R

```diff
@@ -0,0 +1,57 @@
+.getPreferenceScoreDist <- function(connection = NULL, 
+                                    preferenceScoreDist = NULL,
+                                    targetId,
+                                    comparatorId,
+                                    outcomeId,
+                                    databaseId,
+                                    analysisId) {
+  if (!is.null(connection)) {
+  sql <-   "SELECT *
+            FROM @schema.@preferenceScoreDist
+            WHERE target_id in (@targetId)
+            	and comparator_id in (@comparatorId)
+            	and outcome_id in (@outcomeId)
+            	and database_id in (@databaseId)
+            	and analysis_id in (@analysisId)"
+  
+  schema <- connection@schema
+  sql <- SqlRender::renderSql(sql = sql)
+  sql <- SqlRender::translate(sql, targetDialect = connection@dbms)$sql
+  preferenceScoreDist <- DatabaseConnector::querySql(connection, sql)
+  colnames(preferenceScoreDist) <- SqlRender::snakeCaseToCamelCase(colnames(preferenceScoreDist))
```

Comment:
I recommend using snakeCaseToCamelCase = TRUE when calling querySql instead, making this line of code unnecessary.